generated/nifake/session.py

```diff
@@ -955,6 +955,27 @@ def get_a_string_using_python_code(self, a_number):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return a_string_ctype.value.decode(self._encoding)
 
+    @ivi_synchronized
+    def get_a_string_with_specified_maximum_size(self, buffer_size):
+        r'''get_a_string_with_specified_maximum_size
+
+        Returns multiple types.
```

Comment:
Is this right? Seems like a bug in the generator (if so, should be addressed in its own PR)